desktop/src/main/resources/bisq.css

```diff
@@ -585,6 +585,15 @@
     -fx-font-family: "IBM Plex Sans Thin";
 }
 
+#base-amount-text-field-small {
+    -fx-background-color: transparent;
+    -fx-border-style: none;
+    -fx-text-fill: -bisq-white;
+    -fx-highlight-fill: -bisq-green;
+    -fx-font-size: 4em;
```

Comment:
I suggest something smaller. Just tried creating an offer with VND (Vietnam) and it doesn't fit in the box due to the two decimal places. I tried with 3em, it would likely fit most cases.

Another consideration is that such low value currencies likely don't use decimal places.